commit df4315adc54b825debb55068e6bcf5dbb7545f14 (refs/changes/62/1662/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-03-14T10:17:05-07:00 (2 years, 7 months ago)
    
    DOCKER-1016 Upgrade docker to bunyan 1.8.8 to better handle long log lines

diff --git a/package.json b/package.json
index 6c6a822..d75c9e5 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "assert-plus": "1.0.0",
     "async": "0.9.0",
     "docker-registry-client": "3.2.6",
-    "bunyan": "1.8.5",
+    "bunyan": "1.8.8",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
     "fwrule": "git+https://github.com/joyent/sdc-fwrule.git#d1174be",
     "imgmanifest": "git+https://github.com/joyent/node-imgmanifest.git#91e5d80",
